pandas/tseries/tests/test_timeseries.py

```diff
@@ -4441,6 +4441,15 @@ def test_nat_operations(self):
         self.assertEqual(s.min(), exp)
         self.assertEqual(s.max(), exp)
 
+    def test_round_nat(self):
```

Comment:
I'm sorry, Jeff. I didn't get what do you mean. In the previous test, I tested `round`, `ceil`, `floor` methods of the `DatetimeProperties` obj.
Could you please clarify for what object should I test the methods?